adam-core/src/main/scala/org/bdgenomics/adam/converters/FastqRecordConverter.scala

```diff
@@ -86,19 +89,23 @@ class FastqRecordConverter extends Serializable with Logging {
     assert(readSequence.length == readQualities.length,
       "Read " + readName + " has different sequence and qual length.")
 
-    AlignmentRecord.newBuilder()
-      .setReadName(readName)
-      .setSequence(readSequence)
-      .setQual(readQualities)
-      .setReadPaired(false)
-      .setProperPair(null)
-      .setFirstOfPair(null)
-      .setSecondOfPair(null)
-      .setReadNegativeStrand(null)
-      .setMateNegativeStrand(null)
-      .setPrimaryAlignment(null)
-      .setSecondaryAlignment(null)
-      .setSupplementaryAlignment(null)
-      .build()
+    val builder =
```

Comment:
Nit: Would it be OK with you to put `val builder =` and `AlignmentRecord.newBuilder()` on the same line? Splitting them across two lines doesn't improve readability (IMO) and causes a large spurious diff (because the spacing of the following lines is changed).